task/source-build/0.1/source-build.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: source-build
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+spec:
+  description: Source image build.
+  params:
+    - name: BINARY_IMAGE
+      description: The corresponding binary image based on which to generate source image.
+      type: string
+    - name: BASE_IMAGES
+      description: >-
+        Base images used to build the binary image. Each image per line in the same order of FROM
+        instructions specified in a multistage Dockerfile. Default to an empty string, which means
+        to skip handling a base image.
+      type: string
+      default: ""
+  results:
+    - name: BUILD_RESULT
+      description: Build result.
+    - name: SOURCE_IMAGE_URL
+      description: The source image url.
```

Comment:
I think it should include a additional result here named `SOURCE_IMAGE_DIGEST` to match the Tekton Chains hinting conventions. This will allow EC to verify the digest matches the expected value, and also allow chains to sign the source image.

Reply:
Pushed a commit to output the digest. PTAL.